tests/unit/test_row_noise.py

```diff
@@ -35,35 +36,80 @@ def dummy_data() -> pd.DataFrame:
     )
 
 
-def test_omit_row(dummy_data: pd.DataFrame, fuzzy_checker: FuzzyChecker) -> None:
+@pytest.mark.parametrize("expected_noise", ["default", 0.01, 0.02, 0.05])
```

Comment:
I considered using a numeric value instead of the string "default" because mixing types is kind of gross but I think this approach has the advantage of readability which I value more in a small test.